src/interactions/services/reactions.py

```diff
@@ -0,0 +1,19 @@
+from wagtail.models import Page
+
+from interactions.models import Reaction, ReactionType
+from news.models import NewsPage
+from user.models import User
+
+
+def manage_reaction(user: User, page: Page, reaction_type: str):
+
+    if not isinstance(page, NewsPage):
+        raise ValueError("The page must be a NewsPage.")
+
+    if not reaction_type or reaction_type not in ReactionType.values:
+        Reaction.objects.filter(user=user, page=page).delete()
+        return None
```

Comment:
I think there are 2 different scenarios that we want to manage here:
- If the `reaction_type` is None we want to clear the reaction from the database
- If the `reaction_type` is not an allowed Reaction type value raise an issue

```suggestion
    if reaction_type is None:
        Reaction.objects.filter(user=user, page=page).delete()
        return None

    if reaction_type not in ReactionType.values:
        raise ValueError(f"{reaction_type} is not a valid ReactionType")
```